proto/src/determined/api/v1/agent.proto

```diff
@@ -47,6 +47,8 @@ message GetAgentsResponse {
 message GetAgentRequest {
   // The id of the agent.
   string agent_id = 1;
+  // The resource manager for the request.
+  string resource_manager = 2;
```

Comment:
Is this optional? Or required?

Is it optional if you just have a single resource manager?

Can we mark optional fields with ```optional string resource_manager = 2;```

Reply:
I mean, it can always be blank "" -- there will have to be some logic in the router that checks that checks these cases.